CIP-XXXX/README.md

```diff
@@ -0,0 +1,141 @@
+               / unreg_drep_cert
+               / update_drep_cert
+               / stake_registration_cert
+               / stake_deregistration_cert
+               / stake_delegation_cert
+               / vote_delegation_cert
+
+; New orthogonal certificates
+stake_registration_cert = (N, stake_credential, coin)
+stake_deregistration_cert = (N+1, stake_credential, coin)
+stake_delegation_cert = (N+2, stake_credential, pool_keyhash)
+vote_delegation_cert = (N+3, stake_credential, drep)
+
+; Others remain unchanged
+```
+
+### Semantics
+The orthogonal certificates creates a clear seperation of concerns, where each certificate has a clear, single responsibility:
+
+- `stake_registration_cert` - Registers a specified stake credential
+- `stake_deregistration_cert` - Deregisters a specified stake credential
+- `stake_delegation_cert` - delegates the stake controlled by a specified stake credential to a specified pool
+- `vote_delegation_cert` - delegates the voting power of a specified stake credential to a specified DRep
+
+### Handling Multiple Certificates
+To perform multiple operations, such as registration and stake delegation, multiple certificates would be included in a single transaction. The ledger would process certificates in the order that they are provided. If a stake delegation occurs before the registration of the specified stake credential, that delegation is invalid.
+
+
+### Example Usage
+To achieve the functionality of the current certificates:
+
+
+| Existing Certificate | Orthogonal Certificate Equivalent |
+| -------- | -------- |
+| `reg_cert`              | `stake_registration_cert`       |
+| `unreg_cert`            | `stake_deregistration_cert`     |
+| `stake_delegation`      | `stake_delegation_cert`         |
+| `stake_vote_deleg_cert` | `stake_delegation_cert` +  `vote_delegation_cert`   |
+| `stake_reg_deleg_cert`      | `stake_registration_cert` + `stake_delegation_cert`   |
+| `stake_vote_reg_deleg_cert` | `stake_registration_cert` + `stake_delegation_cert` + `vote_delegation_cert`  |
+
+
+## Rationale: how does this CIP achieve its goals?
+
+The orthogonal certificate provides **Conceptual Clarity** which has several benefits:
+1. **Usage:** The transaction is the fundamental interface for Cardano. It should be as simple as possible for users.
+2. **Documentation:** Each certificate is self-describing and encapsulates a single action, making documentation much easier to write.
+3. **Implementation:** Implementing the ledger should be as simple as possible–without losing features or security–to reduce the risk of an incorrect implementation resulting in a fork.
+
+While composite actions would require multiple certificates, multiple small certificates would only add a couple of bytes when compared to the composite certificates.
+
+## Path to Active
+
+### Acceptance Criteria
+
+- [ ]Acceptance of the Ledger team, as well as alternative node implementation maintainers (Amaru, Dingo, and others)
```

Comment:
```suggestion
- [ ] Acceptance of the Haskell Ledger team, as well as alternative node implementation maintainers (Amaru, Dingo, and others)
```

Reply:
Well, it really only requires implementation from the `cardano-ledger`. Since the haskell node is the majority of the network (as it's the only mainnet node currently), if it were implemented there, the rest of the nodes would be required to implement the change.

I added the rest of the nodes in an effort to include them in the decision process on this change (although it's a rather minor one) and, hopefully, to begin to include them in the decision process of future changes.

Reply:
understood & looks great the way you have it after https://github.com/cardano-foundation/CIPs/pull/1021/commits/e893db97f89766f1130360f6ea848479512df40e ⭐